### PR TITLE
workflows: fixes author's review submissions

### PR DESCRIPTION
**Pull request by Panos512**
* Fixes a bug that didn't allow reviewing author's submissions.
* Accept button of workflow's detailed view now has correct styling.

Signed-off-by: Panos Paparrigopoulos <panos.paparrigopoulos@cern.ch>

### CODE REVIEW

**Comment by Panos512** [comment]
WIP Because:

- [ ]  Research fields don't pass to the form.

File: inspirehep/modules/authors/views/holdingpen.py [hunk 1]
```diff
@@ -314,16 +314,23 @@ def newreview():
     objectid = request.values.get('objectid', 0, type=int)
     if not objectid:
         abort(400)
-    workflow_metadata.extra_data['is-update'] = True
+
     workflow_metadata = WorkflowUIRecord.get_record(objectid)['metadata']
 
+    # Initialise extra_data
```
Comment (david-caro) [inline inspirehep/modules/authors/views/holdingpen.py]: (I mean that I have this fix also on my branch, just swapping the lines, without having to initialize it)

Comment (david-caro) [inline inspirehep/modules/authors/views/holdingpen.py]: No need, I can rebase on top of this (the smaller the pr the better). If you were getting it empty, maybe we should do the extra check and initialize only if not there :/, something like:
```
workflow_metadata['extra_data'] = workflow_metadata.get('extra_data', {})
```

Comment (Panos512) [inline inspirehep/modules/authors/views/holdingpen.py]: done 😉

**Comment by david-caro** [comment]
This shows up other errors down the pipeline, but by itself fixes what it should :+1:

**Pull request merged**